src/elm/clinical.ts

```diff
@@ -93,19 +97,19 @@ export class InValueSet extends Expression {
     }
     const valueset = await this.valueset.execute(ctx);
```

Comment:
This exec function should be reworked to work in a very similar way to `AnyInValueSet` I /think/ it makes sense that after executing the code and valueset expression the valueset should be ensured to be a ValueSet and exist. Then the code existing and checking for matches should be done. AnyInValueSet already does this and it works fine.